src/clients/UBAClient/UBAClientUtilities.ts

```diff
@@ -116,13 +41,6 @@ function omitDefaultKeys<T>(obj: Record<string, T>): Record<string, T> {
   }, {});
 }
 
-/**
- * Parses a UBAParsedConfigType into a UBAFeeConfig
- * @param chainId The chain ID to parse the UBA config for
- * @param tokenSymbol The token symbol to parse the UBA config for
- * @param ubaConfig The UBAParsedConfigType to parse
- * @returns The parsed UBAFeeConfig
- */
```

Comment:
Is it intended to remove this docstring? It seems valid given the function implementation.

Reply:
Ah seems like a merge conflict will restore